### PR TITLE
Update response/request format and sample UI (remaining components)

### PR DESCRIPTION
**Pull request by DonOmalVindula**
## Purpose
Implements a new UI in place of the tabs UI that is currently being used to represent the response/request format and their samples in on-prem docs site.

## Goals
This PR is a continuation of the new admonition UI adoption and replaces the existing tab-based UI with a more intuitive block-based UI in **Develop** and **References** sections  and fixes the issue mentioned below.

Resolves wso2/product-is#11847

## Approach

1. For this purpose, MkDocs Material Admonition UI component were used.

2. `Tabs` UI components in above-mentioned places were replaced by `Admonition` UI as the parent and each section of Request/Response Format and Request/Response Sample were included as the contents inside the Admonition as child elements separated with a horizontal rule.

3. `theme.css` file was used to override the default properties of `Admonitions` to make it look in line with the design of on-prem docs site.

image

## Test environment

- mkdocs - 1.1.2
- mkdocs-material - 4.1.1
- mkdocs-material-extensions - 1.0.1
- pymdown-extensions - 5.0

### CODE REVIEW

File: en/docs/deploy/security/configure-transport-level-security.md [hunk 1]
```diff
@@ -139,13 +139,14 @@ name in the `deployment.toml` file.
 2.  Add a new server name by adding the following
     property (under the relevant Tomcat connector configuration):
 
-    ``` toml
+    ```
     [transport.https.properties]
     server="WSO2 WSO2 IS server"
-
```
Comment (dewniMW) [inline en/docs/deploy/security/configure-transport-level-security.md:145]: why is this line removed?

Comment (DonOmalVindula) [reply en/docs/deploy/security/configure-transport-level-security.md:145]: Because having the empty line in between made the code snippet appear in a much larger font. To resolve that, the line was removed

File: en/docs/references/concepts/authentication/hybrid-client-profile.md [hunk 2]
```diff
@@ -3,9 +3,10 @@
 Specifying any of the following ` response_type ` values
 in an authorization request selects the hybrid flow for authentication:
 
-- [code token](#code-token)
-- [code id_token](#code-id_token)
-- [code id_token token](#code-id_token-token)
+- [OpenID Connect Hybrid Flow](#openid-connect-hybrid-flow)
```
Comment (dewniMW) [inline en/docs/references/concepts/authentication/hybrid-client-profile.md:6]: why is this added?

File: en/docs/references/concepts/authentication/hybrid-client-profile.md [hunk 3]
```diff
@@ -126,13 +134,20 @@ The decrypted ` id token ` is shown below.
...
+```
+{
+    "access_token":"1940a308-d492–3660-a9f8–46723cc582e9",
+    "refresh_token":"6b96cc3a-00da-3d7d-acd1–5aaf76dcd9d4",
+    "scope":"openid",
+    "id_token":"eyJ4NXQiOiJOVEF4Wm1NeE5ETXlaRGczTVRVMVpHTTBNekV6T0RKaFpXSTRORE5sWkRVMU9HRmtOakZpTVEiLCJraWQiOiJOVEF4Wm1NeE5ETXlaRGczTVRVMVpHTTBNekV6T0RKaFpXSTRORE5sWkRVMU9HRmtOakZpTVEiLCJhbGciOiJSUzI1NiJ9.eyJhdF9oYXNoIjoiSnJaWTlNdFlWRUlJSlV4LUREQm13dyIsInN1YiI6ImFkbWluIiwiYXVkIjpbIm5jemJnNW01eHh0NnRQNFVNWndCNlB0UW9Rb2EiXSwiYXpwIjoibmN6Ymc1bTV4eHQ2dFA0VU1ad0I2UHRRb1FvYSIsImlzcyI6Imh0dHBzOlwvXC9sb2NhbGhvc3Q6OTQ0M1wvb2F1dGgyXC90b2tlbiIsImV4cCI6MTUxMDgzMjI3MSwibm9uY2UiOiJhc2QiLCJpYXQiOjE1MTA4MzIyNjd9.jAGLp8FFdIyFi4ZmvRPX9hVu8NbLVL2iM1895UNrS7wqgl2PCi7zHnvBoOYkbsxxMYGoVepFNzz7hHbk-kuzq_kBoBsZK2Ucbv0hUkwiEkigLy6hpGm-mqXjai3cjlJevWOVcZbMhkEyRlsZtdUG0RCzteT7emAuZLFm5zfMpq1h5JsVRGjK_6fQbHhB2Svkl_kV_ctAD8_kymASGEjRGnwGW5np4uBI0NPYMDTvrl8N9i6yfUVD9-y7rL9Gtrq9hK28Swj5Szvv_c1IX8wYBP-p8gu2cBpGIulIq-OkbfCUh-rrbh96relOaKwKwk0g7nST6o6wZTAwaicNQBYHYw",
```
Comment (dewniMW) [inline en/docs/references/concepts/authentication/hybrid-client-profile.md:142]: shall we indent this to align with other key-value pairs?

File: en/docs/references/concepts/authorization/uma-permissions-endpoint.md [hunk 4]
```diff
@@ -0,0 +1,100 @@
+# Permissions Endpoint
```
Comment (dewniMW) [inline en/docs/references/concepts/authorization/uma-permissions-endpoint.md:1]: this file is not needed. we have renamed the file to uma-permission-endpoint.md

File: en/docs/references/concepts/authorization/uma-permissions-endpoint.md [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+    **Failed Response**
+    ``` xml
+    HTTP/1.1          400     Bad    Request
+    Content-Type:  application/json
+    ...
+    {
+        "error":"invalid_resource_id",
+        "error_description":"Permission request failed with bad resource ID."
+    }
+    ```
```
Comment (dewniMW) [inline en/docs/references/concepts/authorization/uma-permissions-endpoint.md:88]: however, this change should be done to the renamed file

**Review by dewniMW — APPROVED** [review]
